frontend/package.json

```diff
@@ -6,8 +6,12 @@
     "serve": "vite --host 0.0.0.0",
     "preview": "vite preview --host 0.0.0.0 --port 3000",
     "build": "vite build",
-    "lint": "vue-cli-service lint",
-    "lint-check": "vue-cli-service lint --no-fix",
+    "lint": "npm run lint:eslint && npm run lint:prettier",
+    "lint:eslint": "vue-cli-service lint",
+    "lint:prettier": "prettier --write --ignore-path .gitignore **/*.{css,scss,json,md}",
+    "lint:check": "npm run lint:check:eslint && npm run lint:check:prettier",
+    "lint:check:eslint": "vue-cli-service lint --no-fix",
+    "lint:check:prettier": "prettier --write --ignore-path .gitignore **/*.{css,scss,json,md}",
```

Comment:
This is the same exact command as without :check. Something can't be right here.

Reply:
Fixed in b1acdb3